Azure/sonic-buildimage
# [mellanox] Use 'mlxfwmanager -l' for extracting available firmware version from FW images (Add support for new FW version schema)

Signed-off-by: Andriy Yurkiv <ayurkiv@nvidia.com>



**- Why I did it**
Mellanox FW version can be from different schema (some might be used for internal use but also for public)
The new FW version schema can be based on ‘_’ as delimiter as well as suffix of ‘-xxx’
**- How I did it**
Read the FW version using ‘mlxfwmanager -l’ to extract the FW version
**- How to verify it**
Install new FW version with ‘new’ format via the /usr/bin/mlnx-fw-upgrade.sh. Verify the flow of auto FW upgrade is working as expected after reboot
**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
